TaskManagement/project/views.py

```diff
@@ -19,9 +19,12 @@ class CreateProjectView(LoginRequiredMixin,CreateView):
 
 
     def get_form(self, *args, **kwargs):
-        form = super(CreateProjectView, self).get_form(*args, **kwargs)
-        form.fields['project_lead'].queryset = User.objects.filter(is_superuser=False)
-        return form
+        try:
+            form = super(CreateProjectView, self).get_form(*args, **kwargs)
+            form.fields['project_lead'].queryset = User.objects.filter(is_superuser=False)
+            return form
+        except Exception as error:
+            raise error
```

Comment:
please review what does exception and raise does. this is not at all the right way